### PR TITLE
Add Chromium-GOST v.103.0.5060.134

### PR DESCRIPTION
**Pull request by dvorobiev**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by aaronliu0130** [comment]
Repository only has 30 watches, not notable enough.

File: Casks/chromium-gost.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "chromium-gost" do
+  version "103.0.5060.134"
+  sha256 "f7bcca0f328fae408773700dae023aa036e909d48022157b0c6627b860a1ea78"
+
+  url "https://github.com/deemru/Chromium-Gost/releases/download/#{version}/chromium-gost-103.0.5060.134-macos-amd64.tar.bz2"
+  name "Chromium-Gost"
+  desc "Браузер Chromium с поддержкой алгоритмов ГОСТ"
```
Comment (aaronliu0130) [inline Casks/chromium-gost.rb:7]: I think this should be translated

Comment (dvorobiev) [reply Casks/chromium-gost.rb]: Yes, of course, ✅

**Review by aaronliu0130 — COMMENTED** [review]
I don't have write access so my review will do nothing, but, this software and repo [doesn't seem notable enough.](https://docs.brew.sh/Acceptable-Casks#rejected-casks:~:text=The%20Cask%20is%20for%20an%20app,30%20forks%2C%2030%20watchers%2C%2075%20stars).)

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @dvorobiev.

Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

**Comment by miccal** [comment]
Thank you @dvorobiev.

**Review by miccal — APPROVED** [review]

**Pull request merged**